### DISCUSSION TITLE
Issue updating hyperhdr v17 to v18

### DISCUSSION THREAD

**Discussion by Steeve6885** [awawa-dev/HyperHDR#349] [Q&A]
Hello everyone,

today I try to update my hyperhdr to last update v18 but when it’s done the service look’s not working or even Launch. 
updating with follow of ssh command :

sudo apt remove hyperhdr
 cd /tmp
 wget https://github.com/awawa-dev/HyperHDR/releases/download/v18.0.0.0/HyperHDR-18.0.0.0-Linux-armv7l.deb
 sudo apt install ./HyperHDR-18.0.0.0-Linux-armv7l.deb

reboot the system.

This work perfect for v17.0.0.0 but not for v18.0.0.0 I don’t even acces the webpage hyperhdr for configuration. 

If i did a Fresh install using v18.0.0.0 image will it work ?

**Comment by awawa-dev** [comment]
Hi
Probably you are running not updated Debian Buster based OS and installers are built for Bullseye. Run ```ldd -version``` to check it.
